### DISCUSSION TITLE
Showing top line only when it contains any content.

### DISCUSSION THREAD

**Discussion by ftpd** [starship/starship#2141] [Ideas]
Hi. I would like to have multiline prompt only when top line contains any usable information (like git branch, git status, kubernets context etc). If not, only the 'bottom' part of my prompt (basically directory and character) should be visible.

Is this possible? Or can be? ;-)

**Comment by mickimnet** [comment]
Normally, you could/should(?) do it with [conditional format strings](https://starship.rs/config/#conditional-format-strings), but currently there is a bug (#1855) in regards to the `format` section why it won't work (ATM).
